cmd/awsDataGather/main.go

```diff
@@ -6,7 +6,7 @@ import (
 	"os"
```

Comment:
Although it is a good piece of code. aws-data-gather is not required.Please remove

Reply:
Updated in the latest commit